src/builds/builds.controller.ts

```diff
@@ -67,8 +68,8 @@ export class BuildsController {
   @ApiSecurity('api_key')
   @ApiBearerAuth()
   @UseGuards(MixedGuard)
-  stop(@Param('id', new ParseUUIDPipe()) id: string): Promise<BuildDto> {
-    return this.buildsService.stop(id);
+  update(@Param('id', new ParseUUIDPipe()) id: string, @Body() modifyBuildDto: ModifyBuildDto): Promise<BuildDto> {
```

Comment:
Just realized, this will need changes in all language SDK. They need to pass {"isRunning":false} while sending this request to this controller. Currently, all SDKs are not sending anything in the body to stop the test run. I tested with Java SDK and passing this body stops the build properly.
I think we should modify all SDK as patch should be a generic controller method and the body should specify what should be patched (isRunning in this case). I can take care of Java, Python, and Javascript SDK.

Reply:
@pashidlos  If we want to implement PATCH properly as per RFC, we have to refactor this controller method and all the **_consumers_** to call like below format

PATCH /builds/<id>
[
  { "op": "replace", "path": "/isRunning", "value": false},
  { "op": "replace", "path": "/ciBuildId", "value": "newValue"}
]

Let me know what you think if we can proceed with this proper implementation across the board?

Reply:
@suratdas good catch!
lot's then keep `stop` endpoint for backward compatibility and mark it as deprecated 
I'll create tasks to update all SDKs after we merge this PR

Reply:
I modified controller code a little bit and now SDK will not break. Tested with Java SDK. All look good now. Please review. 
Later, we can address issue number 199 that I logged earlier.